schemas/yaml/Lexeme.yml

```diff
@@ -243,6 +243,16 @@ properties:
     format: uri
     description: A URL where a JSON representation of this lexeme may be retrieved
 
+  usages:
+    title:       Usages
+    type:        array
+    uniqueItems: true
+    description: A list of MultiLangStrings giving information about this lexemes social usage, regional information, register, dialect, and/or connotations. Common values might be archaic, colloquial, formal, positive, negative, etc.
```

Comment:
`lexemes` > `lexeme's`

Let's also put double quotes around each of the example values. (You'll need to wrap the entire `description` field in single quotes.)